libs/blocks/modal/modal.js

```diff
@@ -280,7 +280,7 @@ export function getHashParams(hashStr) {
 
 export function delayedModal(el) {
   const { hash, delay } = getHashParams(el?.dataset.modalHash);
-  if (delay === undefined || !hash) return false;
+  if (delay === undefined || !hash || !window.matchMedia('(min-width: 1200px)').matches) return false;
```

Comment:
let's give a name to the media query to make it easier to understand.
in merch.js we are using `const isMobile = window.matchMedia('(max-width: 1199px)').matches;`

Reply:
Done 👍